src/app/review-item/components/question/question.component.html

```diff
@@ -1,10 +1,12 @@
-<h4>{{question?.content}}</h4>
+<h4>{{question?.content}}<span *ngIf="question?.hint"><mat-icon [matTooltip]="question?.hint">help_outline</mat-icon></span></h4>
 <div *ngIf="question?.info">
-  <p>{{question?.info}}</p>
+  <p [innerHTML]="question?.info"></p>
 </div>
 <form *ngIf="form" [formGroup]="form">
   <mat-radio-group formControlName="option">
-    <mat-radio-button *ngFor="let option of question?.options" [value]="option" [checked]="false">{{option.text}}</mat-radio-button>
+    <mat-radio-button *ngFor="let option of this.options" [value]="option" [checked]="false" [matTooltip]="option.tooltip">
```

Comment:
Shouldn't we also have a small icon here to signalize that a tooltip exists?

Reply:
Isn't this screen supposed to be changed entirely according to Daniels mocks?

Reply:
Yeah, you're right. But the mocks also include the icon. :P 
Are you on the re-design already?